cypress/page/products-list.page.ts

```diff
@@ -3,8 +3,10 @@ class ProductsList {
     private proceedToCheckoutButton: string;
 
     constructor(){
-        this.addToChartButton = "#center_column a.button.ajax_add_to_cart_button.btn.btn-default";
-        this.proceedToCheckoutButton = "[style*='display: block;'] .button-container > a";
+        // Just get a span that's inside in a ajax_add_to_cart class button.
```

Comment:
the justification is only required when you are not agree with one suggestion, please remove the comments 😉

Reply:
I understood that the selectors must work even if software got changes. The tittle attribute is susceptible to changes right? That means if the tittle changes the selector won't work...

Reply:
tag attributes are less likely to change over time than CSS classes, also using attributes improves the readability of the code.